test/unit/presenters/homepage_presenter_test.rb

```diff
@@ -0,0 +1,64 @@
+require "test_helper"
+
+class HomepagePresenterTest < ActiveSupport::TestCase
+  setup do
+    @content_item = {
```

Comment:
@mtaylorgds @syed-ali-tw 

We should really update the homepage schema example in publishing API so that we can use that example in the frontend tests. Could you create a card on your backlog for that?

Reply:
There's no frontend example [here](https://github.com/alphagov/publishing-api/tree/main/content_schemas/examples/homepage)

Reply:
for tracking, here is the [trello](https://trello.com/c/QwQuxFIC/303-document-popular-links-schema-in-devdocs) card we would be using to get example schema work done. the ticket will be more refined in coming days.